.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build generator-symphony NodeJs project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs-or-python
+
+name: Build
```

Comment:
let's maybe prepare the publish workflow too (even if it will fail because of the missing creds)
can be a separated PR if you prefer too

Reply:
Please review https://github.com/finos/generator-symphony/pull/169/commits/3e0166071c7824d8c898f0cb0f499511a541373c